recipe/meta.yaml

```diff
@@ -33,6 +33,7 @@ requirements:
     - r-pbdzmq >=0.2_1
     - r-repr >=0.4.99
     - r-uuid
+    - jupyter
```

Comment:
Including a `jupyter` dependency in the kernel undermines the modularity of kernels. The point is that kernels can be loaded into Jupyter without having Jupyter coinstalled in the same environment.